docs/getting-started.md

```diff
@@ -46,7 +46,72 @@ UPDATE package.json (1507 bytes)
 
 #### IMPORTANT: *Scully requires the router to be present in your application, don't forget to add it.*
 
-## @scullyio/init:blog
+## ng g @scullyio/init:blog
+This command will generate a blog module. [more info here](https://github.com/scullyio/scully/blob/master/docs/blog.md)
+
+Once it's generated you can open the default `app.component.html` created by angular-cli and remove the whole placeholder leaving only the router outlet tag `<router-outlet></router-outlet>`
+
+### Home page
+
+Since the default template from angular-cli doesn't ship an entry point for route, it might be confusing to get scully working on the very first shot
+
+```js
+ng g m home --route=home --module=app-routing
+```
+
+This command will magically generate the new home page module plus a new component with a route configured
```

Comment:
Please remove "magically" from this text.

Reply:
but it's magical LOL

will do the changes later today

Reply:
> but it's magical LOL
> 
> will do the changes later today

No rush! 
Thanks for your support...